arch/power/adler32_vmx.c

```
@@ -145,37 +124,36 @@ Z_FORCEINLINE static uint32_t adler32_impl(uint32_t adler, const uint8_t *buf, s
-    // Align buffer
-    size_t align_len = (size_t)MIN(ALIGN_DIFF(buf, 16), len);
-    if (align_len) {
-        vmx_handle_head_or_tail(pair, buf, align_len);
-        done += align_len;
-        /* Rather than rebasing, we can reduce the max sums for the
-         * first round only */
-        n -= align_len;
+    /* Split Adler-32 into component sums, it can be supplied by
+     * the caller sites (e.g. in a PNG file).
+     */
+    uint32_t pair[16] ALIGNED_(16);
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If ALIGNED_ is a macro then please configure it.

(unknownMacro)